pkg/ring/kv/consul/client.go

```diff
@@ -85,7 +84,7 @@ func NewClient(cfg Config, codec codec.Codec) (*Client, error) {
 		return nil, err
 	}
 	c := &Client{
-		kv:    consulMetrics{client.KV()},
```

Comment:
Previously we could see individual `CAS` calls to consul. Now we will only see "CAS loop".